### PR TITLE
AP_OSD: Add option to increase OSD speed and current display resolution (for slow vehicles)

### PR DESCRIPTION
**Pull request by Hwurzburg**
adds OSD_OPTION to add decimal place in speed and current OSD displays for slow vehicles...ie my tugboat only cruises at 1mph and draws 0.9A when doing so....now I can see differences in speed at cruise and full throttle...

tested on vehicle

### CODE REVIEW

**Comment by khancyr** [comment]
The commit title need AP_OSD: append before the resume !

**Comment by Hwurzburg** [comment]
done...thanks

File: libraries/AP_OSD/AP_OSD_Screen.cpp [hunk 1]
```diff
@@ -1044,8 +1049,11 @@ void AP_OSD_Screen::draw_speed_vector(uint8_t x, uint8_t y,Vector2f v, int32_t y
         int32_t interval = 36000 / SYM_ARROW_COUNT;
         arrow = SYM_ARROW_START + ((angle + interval / 2) / interval) % SYM_ARROW_COUNT;
     }
-
+    if (check_option(AP_OSD::OPTION_EXPANDED_DISPLAY)) {
+        backend->write(x, y, false, "%c%3.1f%c", arrow, u_scale(SPEED, v_length), u_icon(SPEED)); 
+    } else {
     backend->write(x, y, false, "%c%3d%c", arrow, (int)u_scale(SPEED, v_length), u_icon(SPEED));
```
Comment (sh83) [inline libraries/AP_OSD/AP_OSD_Screen.cpp]: Formatting... It actually would be nice to format  whole ap_osd with ardupilot-astyle.sh.

Comment (Hwurzburg) [reply libraries/AP_OSD/AP_OSD_Screen.cpp]: fixed...thanks
I will put doing that on my list....no problem...

**Comment by tridge** [comment]
could this be automatic instead? ie. if speed is below 10 then use an extra digit.

**Comment by Hwurzburg** [comment]
good idea, I'll work on that tomorrow....will field size changing be any issue for panel item placement?, or just live with it?

**Comment by Hwurzburg** [comment]
per @tridge suggestion , eliminated option, just made both current and speed displays autorange below a value of 10 to add a decimal place of precision.

**Pull request merged**